rlberry/agents/ucbvi/ucbvi.py

```diff
@@ -3,7 +3,7 @@
 
 import gym.spaces as spaces
 from rlberry.agents import AgentWithSimplePolicy
-from rlberry.agents.ucbvi.utils import update_value_and_get_action, update_value_and_get_action_sd
+from rlberry.agents.ucbvi.utils import update_value_and_get_action, update_value_and_get_action_sd, backward_induction_reward_sd
```

Comment:
The function `backward_induction_reward_sd` must be imported from `rlberry.agents.dynprog.utils`, no? This should fix the tests.